spec.md

```diff
@@ -202,12 +202,12 @@ attributes designated as REQUIRED and MAY include one or more OPTIONAL context
 attributes.
 
 These attributes, while descriptive of the event, are designed such that they
-can be serialized independent of the event data. This allows for them to be
-inspected at the destination without having to deserialize the event data.
+can be serialized independent of the data payload. This allows for them to be
+inspected at the destination without having to deserialize the data payload.
```

Comment:
I think `event data` referred to an abstract notion (attributes of event vs event data). Removing "event" does not improve this paragraph IMO.

Either:
* Keep it as it is
* Name it `event data payload`